scripts/travis-setup.sh

```diff
@@ -1,37 +1,10 @@
 #!/bin/bash
 set -euo pipefail
 
-SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
-
-if [[ $TRAVIS_OS_NAME = "linux" ]]
-then
-    tag=Linux
-else
-    tag=MacOSX
-fi
-echo $(which python)
+[[ $TRAVIS_OS_NAME = "linux" ]] && tag=Linux || tag=MacOSX
 curl -O https://repo.continuum.io/miniconda/Miniconda3-latest-${tag}-x86_64.sh
 sudo bash Miniconda3-latest-${tag}-x86_64.sh -b -p /anaconda
 sudo chown -R $USER /anaconda
 export PATH=/anaconda/bin:$PATH
 conda install -y --file scripts/requirements.txt
-
-# Install bioconda-utils with miniconda version of python
-git clone https://github.com/bioconda/bioconda-utils.git
-(cd bioconda-utils && python setup.py install)
-
-
-if [[ $TRAVIS_OS_NAME != "linux" ]]
-then
-    mkdir -p /anaconda/conda-bld/osx-64 # workaround for bug in current conda
-    mkdir -p /anaconda/conda-bld/linux-64 # workaround for bug in current conda
-    conda index /anaconda/conda-bld/linux-64 /anaconda/conda-bld/osx-64
-
-    # setup bioconda channel
-    conda config --add channels bioconda
-    conda config --add channels r
-    conda config --add channels file://anaconda/conda-bld
-
-    # setup bioconda-utils
-    pip install git+https://github.com/bioconda/bioconda-utils.git
-fi
+pip install git+https://github.com/bioconda/bioconda-utils.git@docker-use-config
```

Comment:
I merged your docker-use-config branch into the master of bioconda-utils. Hence we can remove the branch tag here.